notebooks/en/structured_generation.ipynb

```diff
@@ -0,0 +1,600 @@
+{
```

Comment:
<p>Might be simpler to do this instead :)</p><p> </p><p><em>"Constrained decoding"</em></p><p> </p><p><em>"</em>To force a JSON output, we'll have to use <strong>constrained decoding </strong><em>where</em> we force the LLM to only output tokens that conform to a set of rules called a <strong>grammar</strong>. <em>"</em></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/64/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>